s4e-backend/src/main/java/pl/cyfronet/s4e/controller/WebhooksController.java

```diff
@@ -0,0 +1,37 @@
+package pl.cyfronet.s4e.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.cyfronet.s4e.bean.Product;
+import pl.cyfronet.s4e.bean.Webhook;
+import pl.cyfronet.s4e.service.GeoServerService;
+import pl.cyfronet.s4e.service.ProductService;
+
+
+import static pl.cyfronet.s4e.Constants.API_PREFIX_V1;
+
+@RestController
+@RequestMapping(API_PREFIX_V1)
```

Comment:
Wouldn't it make sense to expose this controller on a different prefix?
In this way it wouldn't be accesible from the outside, i.e. through front-end.